app/code/Magento/Tax/Test/Mftf/ActionGroup/AdminDeleteTaxRateActionGroup.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminDeleteTaxRateActionGroup">
+        <annotations>
+            <description>Goes to the Admin Tax Rate grid page. Deletes the provided Tax Rate Code.</description>
```

Comment:
In my opinion the description doesn't reflect the ActionGroup behavior, additionally we have more actions (filter, selectFirstRow, delete) here than is supposed to.